src/app/global-header/global-header.component.html

```diff
@@ -1,13 +1,31 @@
 <header>
   <div routerLink="/">
-    <img src="assets/images/logo.svg" alt="logo" class="logo">
-    <span class="title">
-      <span class="highlight-title">ULP</span>
-      Registration Portal
-    </span>
-  </div>
+      <div class="row">
+        <div class="col-md-3">
+          <div class="d-flex">
+          <div>
+            <img src="../../../assets/images/headerlogo.png" class="img-fluid" alt="...">
```

Comment:
@Ekta29yadav Please dont use relative path.. add path with starting from assets otherwise it will fail in prod build

Reply:
@itsvick Done